syntax = "proto3";

service EchoService {
  rpc EchoMessage (EchoRequest) returns (EchoResponse);
}

message EchoRequest {
  string message = 1;
}

message EchoResponse {
  string echoed_message = 1;
}
